test/e2e/changedFilesUtil.js

```diff
@@ -29,10 +29,10 @@ function readChangedFiles() {
 /**
  * Filters the list of changed files to include only E2E test files within the 'test/e2e/' directory.
  *
- * @returns {<string[]>} An array of filtered E2E test file paths.
+ * @param {string[]} changedFiles - An array of changed file paths to filter.
```

Comment:
Nit: can we rename to `changedFilesPaths` to reflect the param more?

Reply:
make sense, thank you!!